app/models/company.rb

```diff
@@ -1,3 +1,14 @@
 class Company < ActiveRecord::Base
+  has_many :freebies
+  has_many :devs, through: :freebies
 
+  def give_freebie(dev, item_name, value)
+    Freebie.create(item_name: item_name, value: value,
+                  dev: dev, company: self)
+  end
+
+  def self.oldest_company
+    oldest_founding_year = Company.minimum(:founding_year)
+    Company.where(founding_year: oldest_founding_year)
```

Comment:
`where` returns an array. I believe we want a single record here.

Reply:
I can add .first here and it will be out of array format.